src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -57,9 +57,15 @@ const MerchantsPage = (props: Props) => {
     if (type === 'all') {
       setFilter(sellers);
     } else {
-      const result = sellers.filter(
+      let result = sellers.filter(
         (store: any) => store!.locations.length > 0 && store!.locations[0].city === type
       );
+      if (type === 'Floral Park') {
+        const flushing = sellers.filter(
+          (store: any) => store!.locations.length > 0 && store!.locations[0].city === 'Flushing'
+        );
+        result.push(...flushing);
+      }
```

Comment:
this is good quick fix! We should probably add a borough attribute to the backend. Let's pair on this tomorrow